server/sonar-search/src/main/java/org/sonar/search/EsSettings.java

```diff
@@ -149,6 +149,31 @@ private InetAddress readHost() {
     }
   }
 
+  private String readInitialStateTimeout() {
+    String initialStateTimeout = props.value(ProcessProperties.SEARCH_INITIAL_STATE_TIMEOUT, "30s").toLowerCase().trim();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define the locale to be used in this String operation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1449)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "value" is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2259)